mob_test.go

```diff
@@ -1995,6 +1997,8 @@ func cloneRepository(path, remoteDirectory string) {
 	git("clone", "--origin", "origin", "file://"+remoteDirectory, ".")
 	git("config", "--local", "user.name", name)
 	git("config", "--local", "user.email", name+"@example.com")
+	// see bug #346 changes the output of git status --short
+	git("config", "--local", "status.branch", "true")
```

Comment:
Is now every test using this config? Or should we have a single test that makes use of this config change only?

Reply:
I did it for every test and that was good, as I found some more points where `--short` was used and this option let the tests for it fail.

Reply:
due to all tests having `git("config", "--local", "status.branch", "true")` now, our test-coverage is not better than it was before that. I would prefer if we had just one test that contained the line `git("config", "--local", "status.branch", "true")` to test specifically that case

Reply:
Do you think this is enough? There where multiple Test that failed. I would at least copy all of them then.

Reply:
Did all those tests fail of the same reason? Or did they fail for different reasons?
If they failed for the same reason, I would only pick one

Reply:
They failed because all used git status --short command but they did use it in different methods and for different things